src/asm/lexer.c

```diff
@@ -921,6 +921,31 @@ void lexer_DumpStringExpansions(void)
 	free(stack);
 }
 
+/* Discards an inline comment */
+static void discardInlineComment(void)
+{
+	dbgPrint("Discarding inline comment\n");
+	int c;
+	shiftChars(2);
```

Comment:
Why 2? Shouldn't it be 1 from the `peek(0)` that led here? (Arguably, I'd rather move that `shiftChars` to the calling site, I think it'd be clearer)

Reply:
yes, good catch. i'll move the shiftChars as well, wasn't sure where that was best put